src/DNS_Mgr.cc

```diff
@@ -1403,7 +1403,22 @@ double DNS_Mgr::GetNextTimeout()
 	if ( asyncs_pending == 0 )
 		return -1;
 
-	return run_state::network_time + DNS_TIMEOUT;
+	int nfds = 0;
+	fd_set read_fds, write_fds;
+
+	struct timeval tv;
+	tv.tv_sec = DNS_TIMEOUT;
+	tv.tv_usec = 0;
```

Comment:
Suggested readability tweak: move this block to just before `struct timeval* tvp = ares_timeout(channel, &tv, &tv);`.

Reply:
Good point. That'll avoid allocating it if we don't need it too.